ussd/screens/schema.py

```diff
@@ -57,6 +57,11 @@ class WithItemSchema(Schema):
 class WithDictSchema(Schema):
     with_dict = WithDictField(required=False, default=None)
 
+class PaginationConfigSchema(Schema):
+    ussd_text_limit = fields.Integer(required=False, default=180)
+    more_option = fields.Dict()
```

Comment:
instead of fields.Dict should be https://github.com/ussd-airflow/ussd_engine/blob/master/ussd/screens/fields.py#L5